### DISCUSSION TITLE
how to show percentage of ram instead of the value?

### DISCUSSION THREAD

**Discussion by binhjy** [winsiderss/systeminformer#1677] [General Discussion]
Hi all,
How to show percentage of ram instead of the value?
I'm using the nightly build.
Thanks.

**Comment by Biswa96** [comment]
Where do you want to show the ram percentage? SI shows that in many places.

**Reply by binhjy** [reply]
Hi, at the first screen when open the app, I want it to show the percentage like the CPU
![image](https://user-images.githubusercontent.com/16265532/230261552-6d099b0b-11c3-45a8-b4d5-38681f245f7a.png)

**Comment by dmex** [comment]
![image](https://user-images.githubusercontent.com/1306177/230189802-a29baaa6-7147-482d-b3e2-b87cf58b17b3.png)

**Reply by binhjy** [reply]
Is it possible to show in the first screen when open the app?
![image](https://user-images.githubusercontent.com/16265532/230261687-1bab958e-d1be-46e4-ae9c-613935beb557.png)

**Reply by Biswa96** [reply]
> Is it possible to show in the first screen when open the app?

Do you mean something like the following attached screenshot?

![image](https://user-images.githubusercontent.com/31443074/230778640-3975cc53-734b-4a5e-9ecd-4954079b92cd.png)

**Reply by binhjy** [reply]
> > Is it possible to show in the first screen when open the app?
> 
> Do you mean something like the following attached screenshot?
> 
> ![image](https://user-images.githubusercontent.com/31443074/230778640-3975cc53-734b-4a5e-9ecd-4954079b92cd.png)

great, I see that now.
Is it possible to show in the column as well?
![image](https://user-images.githubusercontent.com/16265532/230778798-1f473939-b8cb-4d7e-93bd-3ccaa176a32a.png)
Also do you know which column to show the physical memory?
![image](https://user-images.githubusercontent.com/16265532/230778862-14083282-f33c-4fd5-b27f-107097aa4980.png)

**Reply by Biswa96** [reply]
> Is it possible to show in the column as well?

I do not know if it is possible to show that percentage value in column header.

> Also do you know which column to show the physical memory?

If you want the physical memory value in the footer like in my screenshot, right click on the footer itself and select customize option.

**Reply by binhjy** [reply]
> If you want the physical memory value in the footer like in my screenshot, right click on the footer itself and select customize option.

No, I mean show physical memory in the column header not footer? Is it possible?

**Reply by Biswa96** [reply]
I am not sure. Please wait for other to comment on.

**Comment by binhjy** [comment]
hi
